jabgui/src/main/java/org/jabref/gui/preferences/preview/PreviewTabViewModel.java

```diff
@@ -501,7 +506,45 @@ public StringProperty sourceTextProperty() {
     public void addBstStyle(Path bstFile) {
         BstPreviewLayout bstPreviewLayout = new BstPreviewLayout(bstFile);
         bstStylesPaths.add(bstFile);
+        manuallyAddedBstPaths.add(bstFile); // Mark as manually added
         availableListProperty().add(bstPreviewLayout);
         chosenListProperty().add(bstPreviewLayout);
     }
+
+    public void removeBstStyle(PreviewLayout layout) {
+        if (!(layout instanceof BstPreviewLayout bstLayout)) {
```

Comment:
no assignment to variable needed.

Why isn't the method parameter dirctly BstPreviewLayout?